core/internal/settings/settings.go

```diff
@@ -249,3 +249,8 @@ func (s *Settings) GetForkFrom() *spb.RunMoment {
 func (s *Settings) GetIdentityTokenFile() string {
 	return s.Proto.IdentityTokenFile.GetValue()
 }
+
+// Mode of console capture. One of "off", "wrap", "redirect".
```

Comment:
Would "wrap" and "redirect" ever be useful in the Go code? How about making this an `IsConsoleCaptureEnabled` method that checks "off" instead?

Reply:
yeah, i was thinking the same, i think it make more sense... let me update this pr